### DISCUSSION TITLE
Can the Chia blockchain database (db) directory .chia/mainnet/ for Windows version be moved to another hard drive?

### DISCUSSION THREAD

**Discussion by RedChimae** [Chia-Network/chia-blockchain#18389] [Support]
Can the Chia blockchain database (db) directory .chia/mainnet/ for Windows version be moved to another hard drive?

As we all know, under the default installation method, the database of Chia blockchain is located in the directory C:\Users<YOURUSERNAME>/.chia/mainnet/db/. Due to the huge size of the database (db), storing it on the system partition will occupy a lot of disk space, which may even slow down the operation system until the disk is full and the computer crashes. Therefore, considering that the database (db) will gradually grow larger, is there a way to specifically transfer it to another hard drive, so that the main body of Chia blockchain can run separately from the database?

**Comment by BrandtH22** [comment]
Hey @RedChimae , yes you can move the blockchain db to another drive. We recommend that you only use fast SSDs to store the blockchain db as having too slow of a drive will lead to your node falling behind sync. 

Instructions on moving the db can be found here:
https://docs.chia.net/faq/#how-do-i-move-the-chia-blockchain-database-to-another-drive

Feel free to jump into our discord server if you run into issues or have any questions:
https://discord.gg/chia

**Reply by RedChimae** [reply]
**Thank you very much! The Chia blockchain database has been successfully moved to another drive, and the full node synchronization has been completed. Here is the migration optimization plan:**

**From Physical NAS to VMware ESXi: Strategic Migration and Storage Optimization for Chia Blockchain Nodes**

    Recently, I successfully migrated the Chia blockchain environment to the VMware ESXi virtual machine platform. This strategic adjustment aims to optimize resource management and scalability to tackle the growing storage demands and challenges posed by the evolving blockchain technology. Prior to this, the Chia blockchain node was constrained by the 180GB optical drive SSD system disk on an HP ProLiant MicroServer Gen8 physical NAS. As the Chia database (db) rapidly expanded, it reached its storage capacity limit within two years, forcing the node synchronization process to halt.

    In response to the current market uncertainties and the ongoing pursuit of stability, I took proactive measures to securely withdraw XCH tokens stored in the Hpool mining pool to my personal Chia wallet, minimizing potential risks and enhancing control over my assets. To reactivate and stabilize the Chia blockchain synchronization function, I meticulously planned and deployed a virtual machine environment based on Windows 10 IoT Enterprise LTSC 2021.

    During the crucial configuration phase of the virtual machine, I emphasized the optimization of storage performance and scalability. To this end, I carefully allocated a 480GB disk1.vmdk virtual hard disk and wisely chose the VMware Paravirtual SCSI controller for its storage. This technical choice significantly enhanced I/O throughput, laying a solid foundation for the efficient and stable operation of the Chia blockchain node.

    It's worth noting that my VMware ESXi host boasts a diversified data storage disk configuration, further bolstering the system's flexibility and reliability. Specifically, the Intel Optane SSD DC P4800X Series is designated for storing virtual machine instances, ensuring efficient VM operation with its exceptional read-write speeds and low latency. The Intel DC P4618 6.4TB Series is dedicated to storing data required by various business software, safeguarding business continuity with its vast capacity and stability. Meanwhile, the Intel SSD DC S4600 Series 1.92TB is ingeniously utilized to house the ever-growing Chia database (db), providing ample capacity and exceptional performance for blockchain data storage and access.

    Currently, this virtual machine environment is operating smoothly and efficiently executing Chia blockchain node synchronization tasks. This successful migration not only definitively addresses the pressing issue of insufficient storage space but also paves the way for ample resource allocation for future blockchain application expansions, marking another significant technological upgrade and strategic deployment in my blockchain operation and maintenance management endeavors.
 
![捕获 Chia blockchain GUI](https://github.com/user-attachments/assets/c3aa8ecd-985b-4307-b6ec-688c1789246b)

**Reply by BrandtH22** [reply]
Thank you for the confirmation and additional information! Since the migration has succeeded and everything is operating as intended I am closing this issue ticket but if issues do arise do not hesitate to reopen it or open a new one.
